factory/core/coreComponents_test.go

```diff
@@ -4,12 +4,17 @@ import (
 	"errors"
 	"testing"
 
+	"github.com/stretchr/testify/require"
```

Comment:
goimports everywhere.

Reply:
This seems to be ok, it's split in 3 categories.